src/db-service.ts

```diff
@@ -9,10 +9,12 @@ export class ValidationService{
   private static columnsForValidation = sql.join(
     [
       [['ivr', 'id'], ['id']],
+      [['ivr', 'status_id'], ['reviewStatusId']],
+      [['ivr', 'created_at'], ['createdAt']],
       [['iv', 'item_id'], ['itemId']],
-      [['iv', 'result'], ['result']],
+      [['iv', 'status_id'], ['validationStatusId']],
+      [['iv', 'result'], ['validationResult']],
       [['ivp', 'name'], ['process']],
```

Comment:
ivp?

Reply:
Ah, it's item-validation-process table, and we join it to show the process name (instead of id)